pkg/exchange/coinbase/exchange.go

```diff
@@ -485,3 +475,93 @@ func (e *Exchange) DefaultFeeRates() types.ExchangeFee {
 		TakerFeeRate: fixedpoint.NewFromFloat(0.4 * 0.01),  // 40 bps = 0.4%
 	}
 }
+
+// ExchangeTradeHistoryService
+// QueryClosedOrders queries closed orders for the given symbol and time range.
+// startTime and endTime are inclusive.
+func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, startTime, endTime time.Time, lastOrderID uint64) ([]types.Order, error) {
+	if lastOrderID > 0 {
+		log.Warning("lastOrderID is not supported for Coinbase, it will be ignored")
+	}
+	cbOrders, err := e.queryOrdersByPagination(ctx, symbol, []string{"done"})
```

Comment:
startTime and endTime are not applied here?

Reply:
Done